x-pack/plugins/task_manager/server/config.ts

```diff
@@ -67,9 +73,17 @@ const requestTimeoutsConfig = schema.object({
 
 export const configSchema = schema.object(
   {
+    active_nodes_lookback: schema.number({
+      defaultValue: DEFAULT_ACTIVE_NODES_LOOK_BACK_S,
+      min: 10,
+    }),
     allow_reading_invalid_state: schema.boolean({ defaultValue: true }),
     /* The number of normal cost tasks that this Kibana instance will run simultaneously */
     capacity: schema.maybe(schema.number({ min: MIN_CAPACITY, max: MAX_CAPACITY })),
+    discovery_interval: schema.number({
+      defaultValue: DEFAULT_DISCOVERY_INTERVAL_MS,
+      min: DEFAULT_DISCOVERY_INTERVAL_MS,
```

Comment:
not sure if we should set a max as well?

Reply:
I could see 1 or 5 minutes as the max, we could allow a smaller `min`, maybe `1s` or so?

Reply:
Updated in https://github.com/elastic/kibana/pull/190934/commits/730d9337a47ed52f57fe171e57bbb55fbfb0f920 to set max of 5 minutes